FreeSurfer/LongitudinalFreeSurferPipeline.sh

```diff
@@ -54,7 +54,7 @@ opts_SetScriptDescription "Runs the Longitudinal FreeSurfer HCP pipeline"
 # Show usage information
 opts_AddMandatory '--subject' 'SubjectID' 'subject' "Subject ID (required)  Used with --path input to create full path to root directory for all sessions"
 opts_AddMandatory '--path' 'StudyFolder' 'path' "Path to subject's data folder (required)  Used with --subject input to create full path to root directory for all outputs generated as path/subject)"
-opts_AddMandatory '--sessions' 'Sessions' 'sessions' "An @ symbol separated list of session (timepoint, visit) IDs (required). Also used to generate full path to each longitudinal session directory"
+opts_AddMandatory '--sessions' 'Sessions' 'sessions' "Comma separated list of session (timepoint, visit) IDs (required). Also used to generate full path to each longitudinal session directory"
```

Comment:
We use `@` for lists in other places in the pipeline, but not `,` as far as I am aware.  @glasserm, opinion?

Reply:
Let's be consistent.

Reply:
fixed a couple commits ago

Reply:
I assume @glasserm meant "please use @", and github's version of the branch still says "comma" and uses `s/,/ /`.

Reply:
ok